TunnelVisionLabs.ReferenceAssemblyAnnotator/EquivalenceComparers+ArrayDimensionEqualityComparer.cs

```diff
@@ -16,7 +16,9 @@ private ArrayDimensionEqualityComparer()
             {
             }
 
+#pragma warning disable CS8614 // In CLI builds: Nullability of reference types in type of parameter 'x' of 'bool ArrayDimensionEqualityComparer.Equals(ArrayDimension x, ArrayDimension y)' doesn't match implicitly implemented member 'bool IEqualityComparer<ArrayDimension>.Equals(ArrayDimension x, ArrayDimension y)'.
             public bool Equals(ArrayDimension x, ArrayDimension y)
```

Comment:
@sharwell ArrayDimension is a struct coming from the Mono.Cecil package. This issue doesn't repro unless RAA is rewriting reference assemblies.